analytics-provider/src/main/java/org/digma/intellij/plugin/analytics/RestAnalyticsProvider.java

```diff
@@ -877,16 +883,22 @@ Call<ResponseBody> setInsightCustomStartTime(
                 "Accept: application/+json",
                 "Content-Type:application/json"
         })
```

Comment:
this code is not backward compatible, what if user updated the plugin but not the backend ?

Reply:
fixed